pymathics/graph/algorithms.py

```diff
@@ -176,23 +176,39 @@ def eval(self, graph, expression, evaluation: Evaluation, options: dict):
 
 class PlanarGraphQ(_NetworkXBuiltin):
     """
+    See <url>https://en.wikipedia.org/wiki/Planar_graph</url>
+
     <dl>
       <dd>PlanarGraphQ[g]
       <dd>Returns True if g is a planar graph and False otherwise.
     </dl>
 
     >> PlanarGraphQ[CycleGraph[4]]
     = True
+
     >> PlanarGraphQ[CompleteGraph[5]]
     = False
+
+    >> PlanarGraphQ[CompleteGraph[4]]
+     = True
+
+    >> PlanarGraphQ[CompleteGraph[5]]
+     = False
+
+    #> PlanarGraphQ[Graph[{}]]
+     = False
+
+    #> PlanarGraphQ["abc"]
+     : Expected a graph at position 1 in PlanarGraphQ[abc].
+     = False
     """
 
     options = DEFAULT_GRAPH_OPTIONS
 
     def eval(self, graph, expression, evaluation: Evaluation, options: dict):
-        "PlanarGraphQ[graph_, OptionsPattern[PlanarGraphQ]]"
+        "%(name)s[graph_, OptionsPattern[PlanarGraphQ]]"
```

Comment:
Personally, I have been shying away from `%(name)s` . The reason is this mixes two levels of jargon: Python's convention inside a docstring, and the homegrown Mathics3 convention which is its own wonkiness.

Here, it is improbable that PlanarGraphQ gets renamed to something else.  So let's try to be go for _clarity_ instead of assumed jargon for a change.